src/scss/patternlib.scss

```diff
@@ -16,6 +16,27 @@
   &__body {
     display: flex;
     flex: 1;
+
```

Comment:
This will potentially affect headings in the example components as well. The selectors may need to be a bit more specific to prevent this

Reply:
@bameyrick I've amended this now, and moved typography headings into an example.